src/features/usage-consolidation-accordion/usage-consolidation-content-resource/usage-consolidation-content-resource.test.js

```diff
@@ -0,0 +1,116 @@
+import { render } from '@testing-library/react';
+
+import Harness from '../../../../test/jest/helpers/harness';
+
+import UsageConsolidationContentResource from './usage-consolidation-content-resource';
+import { costPerUseTypes } from '../../../constants';
+
+const costPerUseData = {
+  data: {
+    [costPerUseTypes.RESOURCE_COST_PER_USE]:{
+      attributes: {
+        analysis: {
+            cost: 1,
+            costPerUse: 1,
+            usage: 1
```

Comment:
The indentations should be 2 spaces and not 4, please check with eslint (for convenience you can add an extension to the IDE you're working in) and fix all these warnings.

Reply:
Implemented the changes in latest commit